shared/teams/team/members/member-row/index.js

```diff
@@ -124,6 +126,13 @@ export const TeamMemberRow = (props: Props) => {
             </ButtonBar>
           </Box>
         )}
+      <Box style={{...globalStyles.flexBoxRow, flexShrink: 1}}>
+        <Icon
```

Comment:
This looks a little small on mobile (designs say 20x20). Also, it might make sense to make it conditional on `props.active || !isMobile`, since the layout is messed up on mobile if the user is reset.
![6f94da54-1c18-451e-979e-c266935fefc6](https://user-images.githubusercontent.com/11968340/36850596-41af4f42-1d35-11e8-8673-b42cfaa03073.jpg)